test/src/tests/Gopkg.lock

```diff
@@ -27,12 +27,12 @@
   version = "v3.2.0"
 
 [[projects]]
-  digest = "1:4cc7748b74453830951f95ddc3fd980387e7c13f77cc33d7cba54cd6b96dad26"
-  name = "github.com/kr/pretty"
+  digest = "1:73770491417b7cb646b28610eec6481187aba7b5c3703da0d42f8d590d016f5f"
+  name = "github.com/gofrs/uuid"
```

Comment:
Hmm looks like `gopkg.toml` wasn't updated.  You might need to update it by hand and rerun `gulp updatetestdep`

Reply:
I have updated the toml, please have a look, thanks